homeassistant/components/plex/media_browser.py

```diff
@@ -102,9 +102,8 @@ def build_item_response(payload):
             try:
                 children_media_class = ITEM_TYPE_MEDIA_CLASS[library_or_section.TYPE]
             except KeyError as err:
-                raise BrowseError(
-                    f"Media not found: {media_content_type} / {media_content_id}"
-                ) from err
+                _LOGGER.debug("Unknown type received: %s", library_or_section.TYPE)
+                raise UnknownMediaType from err
```

Comment:
This should still raise BrowseError as that's what we're catch when calling this function.

Reply:
Good call, ran through this too quickly. Luckily we should never receive this error unless Plex changes their API payloads.

Reply:
If it's an impossible case we can remove it.

Reply:
I'll dig a bit more first and open a separate PR if it makes sense.